.github/CODEOWNERS

```diff
@@ -4,5 +4,5 @@
 
 ## Docs
 
-docs/ @DataDog/baklava
-README.md @DataDog/baklava
+/docs/ @DataDog/baklava @DataDog/rum-mobile
+*.md @DataDog/baklava @DataDog/rum-mobile
```

Comment:
I'm not sure you want to listen to `*.md`
Among others we do update the `CHANGELOG.md` quite regularly, and you might not want to listen on that.